src/redux/books/store.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly consider writing your books actions and reducers as stated in the project requirement 👍🏼 and please use the duck's pattern for directory structuring `/src/redux/books/books.js`, currently this file contains the store content that should be in the path `/src/redux/store.js` 💪🏼 

- The book reducers and actions are:
    - Action types for adding and removing a book.
    - Set initial state.
    - Export action creators.
    - set the book reducers for adding and removing books.